### PR TITLE
logs: init klog flags

### PR DESCRIPTION
**Pull request by Tal-or**
Signed-off-by: Talor Itzhak <titzhak@redhat.com>

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -94,6 +94,7 @@ func main() {
 		Development: true,
 	}
 	opts.BindFlags(flag.CommandLine)
+	klog.InitFlags(nil)
```
Comment (fromanirh) [inline main.go:97]: I'm, not sure the argument is correct

Comment (Tal-or) [reply main.go:97]: Same as in PAO https://github.com/openshift-kni/performance-addon-operators/blob/master/main.go#L89

Comment (fromanirh) [reply main.go:97]: OK, we do what I'd expect inside the function itself: https://github.com/Tal-or/numaresources-operator/blob/387e57987d0485850698e61239c75d1e3adc7f9e/vendor/k8s.io/klog/v2/klog.go#L425

**Pull request merged**